.github/workflows/ci-mingw.yml

```diff
@@ -82,4 +82,4 @@ jobs:
       run: ninja
     - name: Run AVIF Tests
       working-directory: ./build
-      run: ctest -j $Env:NUMBER_OF_PROCESSORS --output-on-failure
```

Comment:
No idea, maybe @kmilos  has an idea? Thx

Reply:
No, sorry...

Reply:
Yup: https://github.com/AOMediaCodec/libavif/actions/runs/8411426945/job/23030997022#step:10:7

This should be just `$NUMBER_OF_PROCESSORS` in case you want to bring it back.